parsers/julia.c

```diff
@@ -44,19 +44,54 @@ typedef enum {
     K_MODULE,
     K_STRUCT,
     K_TYPE,
-    // K_IMPORT,
+    K_UNKNOWN,
     K_NONE
 } JuliaKind;
 
+typedef enum {
+    JULIA_MODULE_IMPORTED,
+    JULIA_MODULE_USING,
+} juliaModuleRole;
+
+typedef enum {
+    JULIA_UNKNOWN_IMPORTED,
+    JULIA_UNKNOWN_USING,
+} juliaUnknownRole;
+
+/*
+*  using X               X = (kind:module, role:using)
+*
+*  using X: a, b         X = (kind:module, role:using)
+*                     a, b = (kind:unknown, role:using, scope:module:X)
+*
+*  import X              X = (kind:module, role:imported)
+*
+*  import X.a, X.b       X = (kind:module, role:imported)
```

Comment:
About the role for X, I think "importedPartially" or something is better than "imported".

Even if c is defined in X, c is not visible in the source where "import X.a, X.b" is written.
"importedPartially" can represent this situation.

If we use "imported" role here, about "X" we cannot distinguish "import X" and "import X.a, X.b" from a tag file.

I'm not sure "importPartially" is the best. "importIndirectly" is another name I can find.